src/app/checkout/step-3/step-3.component.js

```diff
@@ -41,6 +41,7 @@ class Step3Controller {
     this.sessionStorage = $window.sessionStorage
     this.selfReference = this
     this.isBranded = envService.read('isBrandedCheckout')
+    this.datadogRum = datadogRum
```

Comment:
Why did you create a local version of `datadogRum`?

Reply:
It makes it easier to test